pkgs/top-level/all-packages.nix

```diff
@@ -669,11 +669,113 @@ in
 
   albert = libsForQt5.callPackage ../applications/misc/albert {};
 
+  ### APPLICATIONS/TERMINAL-EMULATORS
+
```

Comment:
It might be wiser to add this section before or after another section. As it is it's in the middle of the tools section

Reply:
This file is a particular mess. I will create another pull request in order to cleanup this a bit - after all, I just searched the first occurrence of a terminal emulator and put everything after that.